velox/expression/Expr.cpp

```diff
@@ -1753,9 +1753,11 @@ std::vector<common::Subfield> Expr::extractSubfields() const {
 ExprSet::ExprSet(
     const std::vector<core::TypedExprPtr>& sources,
     core::ExecCtx* execCtx,
-    bool enableConstantFolding)
+    bool enableConstantFolding,
+    bool ignoreConstantFoldError)
```

Comment:
@pramodsatya : Maybe say 'constantFoldErrorWithFailFunction' instead to make the config more clear about what it does.

Reply:
Thanks @aditi-pandit, updated accordingly and added a unit test. Could you please take another look?